test/scalability/placement_test.go

```diff
@@ -0,0 +1,224 @@
+package scalability
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
+	clusterapiv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+	clusterapiv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
+	"open-cluster-management.io/placement/test/integration/util"
+)
+
+const (
+	clusterSetLabel = "cluster.open-cluster-management.io/clusterset"
+	placementLabel  = "cluster.open-cluster-management.io/placement"
```

Comment:
nit: placement label is defined in api repo that you can use. it will be fine if we also define clusterSetLabel in api repo.

Reply:
`PlacementLabel` is only for file `types.go` in version v1alpha1, and is not defined in v1/v1beta1, seems it was removed by some reasons. Can we just use the current fix, and add `PlacementLabel` back to v1 if necessary?

The related files
api\cluster\v1alpha1\types.go
api\cluster\v1\types.go

Reply:
oh, We do not have placement upgraded to v1beta1 yet.